### PR TITLE
Changed EGL settings to support more Tizen devices

### PR DESCRIPTION
**Pull request by bwikbs**
* Use pbuffer surface for off-screen rendering
* Provides an alternative path to eglGetProcAddress func.

### CODE REVIEW

File: shell/platform/tizen/tizen_surface_gl.cc [hunk 1]
```diff
@@ -77,14 +78,127 @@ uint32_t TizenSurfaceGL::OnGetFBO() {
...
+#define GL_FUNC(FunctionName)              \
+  else if (strcmp(name, #FunctionName) == 0) {    \
+    return reinterpret_cast<void*>(FunctionName); \
+  }
 void* TizenSurfaceGL::OnProcResolver(const char* name) {
   auto address = eglGetProcAddress(name);
   if (address != nullptr) {
     return reinterpret_cast<void*>(address);
   }
+  GL_FUNC(eglGetCurrentDisplay)
```
Comment (xuelian-bai) [inline shell/platform/tizen/tizen_surface_gl.cc:90]: I think hardcode is not the best way...

File: shell/platform/tizen/tizen_surface_gl.h [hunk 2]
```diff
@@ -39,6 +39,7 @@ class TizenSurfaceGL : public TizenSurface {
   bool display_valid_ = false;
   EGLDisplay egl_display_ = EGL_NO_DISPLAY;
   EGLSurface egl_surface_ = nullptr;
+  EGLSurface egl_resource_surface_ = nullptr;
```
Comment (seungsoo47) [inline shell/platform/tizen/tizen_surface_gl.h:42]: For consistency, how about changing nullptr to EGL_NO_SURFACE?

File: shell/platform/tizen/tizen_surface_gl.cc (hunk 1 above)
Comment (bwikbs) [reply shell/platform/tizen/tizen_surface_gl.cc:90]: I think so too...but.. I don't have any ideas right now.

File: shell/platform/tizen/tizen_surface_gl.h (hunk 2 above)
Comment (bwikbs) [reply shell/platform/tizen/tizen_surface_gl.h]: done

**Review by seungsoo47 — APPROVED** [review]
LGTM

**Review by bbrto21 — APPROVED** [review]

**Review by seungsoo47 — APPROVED** [review]

**Pull request merged**